src/components/Sublist3r/Sublist3r.tsx

```diff
@@ -0,0 +1,193 @@
+import { useState, useEffect, useCallback } from "react";
+import { Button, Stack, TextInput, Checkbox } from "@mantine/core";
+import { useForm } from "@mantine/form";
+import { CommandHelper } from "../../utils/CommandHelper";
+import ConsoleWrapper from "../ConsoleWrapper/ConsoleWrapper";
+import { SaveOutputToTextFile_v2 } from "../SaveOutputToFile/SaveOutputToTextFile";
+import { RenderComponent } from "../UserGuide/UserGuide";
+import InstallationModal from "../InstallationModal/InstallationModal";
+import { LoadingOverlayAndCancelButton } from "../OverlayAndCancelButton/OverlayAndCancelButton";
+import { checkAllCommandsAvailability } from "../../utils/CommandAvailability";
+
+/**
+ * Represents the form values for the Sublist3r component.
+ */
+interface FormValuesType {
+    targetDomain: string;
+}
+
+/**
+ * The Sublist3r component.
+ * @returns The Sublist3r component.
+ */
+const Sublist3r = () => {
+    // Component state variables
+    const [loading, setLoading] = useState(false); // State variable to indicate loading state
+    const [output, setOutput] = useState(""); // State variable to store the output of the command execution
+    const [allowSave, setAllowSave] = useState(false); // State variable to allow saving of output
+    const [hasSaved, setHasSaved] = useState(false); // State variable to indicate if output has been saved
+    const [isCommandAvailable, setIsCommandAvailable] = useState(false); // State variable to check if the command is available.
+    const [opened, setOpened] = useState(!isCommandAvailable); // State variable that indicates if the modal is opened.
+    const [loadingModal, setLoadingModal] = useState(false); // State variable to indicate loading state of the modal.
+    const [pid, setPid] = useState(""); // State variable to store the process ID of the command execution.
+    const [verboseMode, setVerboseMode] = useState(false); // State variable for verbose mode
+
+    // Component Constants
+    const title = "Sublist3r";
+    const description = "Sublist3r is a subdomain enumeration tool that gathers subdomains for a given domain.";
+    const steps =
+        "Step 1: Provide the target domain to enumerate subdomains.\n" +
+        "Step 2: Start the enumeration process to gather subdomains.\n" +
+        "Step 3: Review the enumerated subdomains for further analysis.\n";
+    const sourceLink = "https://github.com/aboul3la/Sublist3r"; // Link to the source code
+    const tutorial = "https://github.com/aboul3la/Sublist3r/wiki"; // Link to the official documentation/tutorial
```

Comment:
Please leave this empty. I think the tutorial tab is for future development. 
```suggestion
    const tutorial = ""; // Link to the official documentation/tutorial
```